docs/en/about-appium/eol.md

```diff
@@ -0,0 +1,51 @@
+# Appium version 1 End Of Life (EOL) Plan
+
+Appium 1 has been around for quite a while and now is the time to move
+project development forward and to provide even better automation
+experience with Appium version 2.
+
+## Major Changes
+
+The main differences of Appium 1 in comparison to version 2 are the following:
+
+- Platform drivers are now managed separately from the Appium foundation. This means
+  each driver could now be updated/installed/removed separately from the umbrella one.
+- The foundation modules have been moved to the `@appium` monorepo managed by
+  [Lerna](https://github.com/lerna/lerna). These modules are: appium (now `@appium/appium`),
+  appium-base-driver (now `@appium/base-driver`), appium-fake-driver (now `@appium/fake-driver`),
+  appium-support (now `@appium/support`), appium-doctor (now `@appium/doctor`),
+  eslint-config-appium (now `@appium/eslint-config-appium`), appium-test-support (now `@appium/test-support`),
+  appium-gulp-plugins (now `@appium/gulp-plugins`).
+- Added support of plugins
+- Added a CLI to manage drivers and plugins
+- Dropped support of JSONWP capabilities. Only W3C-compatible clients are now supported.
+- The list of supported command line arguments has been modified to exclude driver-specific ones.
+- Drivers compatible to Appium 2 must include a special manifest in their package.json
+
+## Transition Plan
+
+The preliminary date of Appium 2 release is *1st of January 2022*. After this date we won't
```

Comment:
@jlipps for this date?

Reply:
It should be much sooner, like in the next few months

Reply:
however i'm ok if we want a maintenance plan for 1.x that extends beyond 2.0 release a bit

Reply:
We can release 1.22.0 as the final version as 1.x, or 1.23.0?
(At least, we should release 1.22.0 with this 2.0 announcement on the release page to notify this 2.0 update widely I think.)

For now, it seems we can mark this line as tentative...? (and merge this PR to modify this good documentation more?)

Reply:
I think we should say preliminary release date is like Aug 1 2021. and after this date we won't add features to 1.x. but will continue to provide urgent bugfixes to 1.x through Dec 31 2021.

Reply:
let's not commit to any dates until we can commit to dates 😄

Reply:
seems good for me👌